Formula/zookeeper.rb

```diff
@@ -38,7 +38,7 @@ def shim_script(target)
 
   def default_zk_env
     <<-EOS.undent
-      export ZOOCFGDIR="#{etc}/zookeeper"
+        [ -z "$ZOOCFGDIR" ] && export ZOOCFGDIR="#{etc}/zookeeper"
```

Comment:
This doesn't need any added indentation so remove the two spaces at the beginning.